src/scripts/browserDeviceProfile.js

```diff
@@ -580,9 +581,13 @@ export default function (options) {
         if (browser.tizen) {
             hlsInTsVideoAudioCodecs.push('opus');
         }
-        if (!browser.safari) {
-            hlsInFmp4VideoAudioCodecs.push('opus');
-        }
+        hlsInFmp4VideoAudioCodecs.push('opus');
+    }
+
+    if (safariSupportsOpus) {
```

Comment:
Does `canPlayAudioFormat` need updates to exclude Safari?

Reply:
Safari does not report opus as supported with the way we are checking there so no need to exclude there, at least for now.

Reply:
Maybe this should be an `else if` in case that behavior changes for Safari in the future?